webrtc-stats.html

```diff
@@ -917,6 +921,41 @@ <h2>
                   sender side. If no <a>RTCP Receiver Report</a> has been received yet, then return 0.
                 </p>
               </dd>
+              <dt>
+                <dfn>packetsReceivedWithEct1</dfn> of type <span class="idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets received for this <a>SSRC</a> marked with the "ECT(1)" marking.
+                </p>
+              </dd>
+              <dt>
+                <dfn>packetsReceivedWithCe</dfn> of type <span class="idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets received for this <a>SSRC</a> marked with the "CE" marking.
+                </p>
+              </dd>
+              <dt>
+              <dt>
+                <dfn>packetsReportedAsLost</dfn> of type <span class="idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets for which an [[RFC8888]] section 3.1 report has been sent with a zero R bit.
+                  Only reported if support for the "ccfb" feedback mechanism has been negotiated.
+                </p>
```

Comment:
Should we soften the metric from *AsLost to *NotReceived because R={0,1} can occur due to re-ordering. fractionLost and other packetLost metrics already capture the RTP/RTCP metrics.